mantle.php

```diff
@@ -33,10 +33,10 @@
  *
  * @var string
  */
-define( 'MANTLE_BASE_DIR', __DIR__ ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
+defined( 'MANTLE_BASE_DIR' ) || define( 'MANTLE_BASE_DIR', __DIR__ ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
```

Comment:
Why would this be defined already? Any security concerns with allowing someone to override it?